### PR TITLE
Feature/4618 business account more fields

### PR DESCRIPTION
**Pull request by vladimir-8**

### CODE REVIEW

File: src/Form/BusinessAccountType.php [hunk 1]
```diff
@@ -21,21 +21,26 @@
...
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('name', TextType::class, ['label' => 'registration.step.company.name'])
+            ->add('name', TextType::class, [
+                'label' => 'registration.step.company.name'
+            ])
+            ->add('legalName', TextType::class, [
+                'label' => 'registration.step.company.legal_name',
+                'required' => !$this->authorizationChecker->isGranted('ROLE_ADMIN'),
```
Comment (r0xsh) [inline src/Form/BusinessAccountType.php:38]: Maybe they should be hidden for non-admin, no ?

File: src/Form/BusinessAccountType.php [hunk 2]
```diff
@@ -21,21 +21,26 @@
...
+                'empty_data' => ''])
+            ->add('vatNumber', TextType::class, [
+                'label' => 'registration.step.company.vat_number',
+                'required' => !$this->authorizationChecker->isGranted('ROLE_ADMIN'),
```
Comment (r0xsh) [inline src/Form/BusinessAccountType.php:42]: same

**Review by r0xsh — COMMENTED** [review]
Seems good :+1: 
Nice cleanup on the entity :)

File: src/Form/BusinessAccountType.php (hunk 1 above)
Comment (vladimir-8) [reply src/Form/BusinessAccountType.php:38]: No, they are for `business manager` to setup. So an admin can skip them while creating a business account, but a `business manager` must enter them

**Pull request merged**